lib/android-helpers.js

```diff
@@ -29,6 +30,13 @@ helpers.parseJavaVersion = function (stderr) {
   return null;
 };
 
+helpers.isEmulator = function (opts = null) {
```

Comment:
These helpers are not attached to the driver object, so this method is unavailable. I would add it directly to the driver object, and use the driver's copy of `opts`.